src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/form_div_layout.html.twig

```diff
@@ -86,11 +86,11 @@
 {%- endblock choice_widget_options -%}
 
 {%- block checkbox_widget -%}
-    <input type="checkbox" {{ block('widget_attributes') }}{% if value is defined %} value="{{ value }}"{% endif %}{% if checked %} checked="checked"{% endif %} />
+    <input type="checkbox" {{ block('widget_attributes') }}{% if value is defined %} value="{{ value }}"{% endif %}{% if checked %} checked="checked"{% endif %} />&nbsp;
```

Comment:
Did you tried to change the next "{%- ... -%}" by "{% ... -%}" (deleting the first minus sign) instead of &nbsp; ?
I heard that "{%-" is a variant of "{%" that trims white spaces.